relnotes/requestdispatcher.migration.md

```diff
@@ -0,0 +1,8 @@
+# Don't reuse RequestEventDispatcher instances
```

Comment:
##

Reply:
I'd say "## Adapt `RequestEventDispatcher` usage to new `initialise` method: don't store in a pool".